src/Paket.Core/Files/ProjectFile.fs

```diff
@@ -788,16 +796,17 @@ module ProjectFile =
                                 [lowerCondition,createItemGroup !assemblyTargets frameworkAssemblies,true
                                  condition,createItemGroup libFolder.Targets rest,false]
                         )
-                        
+
         // global targets are targets, that are either directly in the /build folder,
         // or, if there is a framework-restriction, specific to the framework(s).
         // (ref https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package#including-msbuild-props-and-targets-in-a-package). 
         let globalTargets, frameworkSpecificTargets =
             if not importTargets then List.empty, List.empty else
-            let sortedTargets = model.TargetsFileFolders |> List.sortBy (fun lib -> lib.Name)
-            sortedTargets
-            |> List.partition (fun lib -> allTargetProfiles = set lib.Targets )
-        
+            let sortedTargets = model.TargetsFileFolders |> List.sortBy (fun lib -> lib.Path)
+            //sortedTargets
+            //|> List.partition (fun lib -> allTargetProfiles = set lib.Targets )
+            ([] : LibFolder list), sortedTargets
```

Comment:
This seems to be some code-path to "simplify" the generated project file. However it seems like it was never actually used.
After removing `runtime` this seems to work again (and breaks some tests).

Should we get rid of this simplification (as it was never used) or should I update the tests?